addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -45,12 +45,10 @@ $-editor-messages-margin-x: 2%;
         opacity: 0.3;
     }
 
-    &.oe_structure.oe_empty, &[data-oe-type=html], .oe_structure.oe_empty {
-        &:empty {
+    &.oe_structure.oe_empty, .oe_structure.oe_empty {
+        &#wrap:empty, &#wrap >.oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
```

Comment:
it break the editor-message on eCommerce category ! 
Please double check !

Reply:
Probably because you removed `&[data-oe-type=html]` @DramixDW, do you know why you did?